runelite-client/src/main/java/net/runelite/client/plugins/cooking/CookingPlugin.java

```diff
@@ -109,31 +118,41 @@ public void onChatMessage(ChatMessage event)
 
 		final String message = event.getMessage();
 
+		if (message.startsWith("You squeeze the grapes into the jug") && config.fermentTimer())
```

Comment:
Since this string is now used in multiple places, it would be worth making it a `static final` variable.